ovh/resource_dbaas_logs_cluster.go

```diff
@@ -273,6 +273,13 @@ func resourceDbaasLogsClusterRead(d *schema.ResourceData, meta interface{}) erro
 
 	serviceName := d.Get("service_name").(string)
 	clusterId := d.Get("cluster_id").(string)
+	if clusterId == "" {
+		var err error
+		clusterId, err = dbaasGetClusterID(config, serviceName)
+		if err != nil {
+			return fmt.Errorf("error retrieving clusterId for %s:\n\t %q", serviceName, err)
+		}
```

Comment:
you should also add this when successfully retrieving the default cluster id: `d.Set("cluster_id", clusterId)`

Reply:
I'm not sure about this one, we don't want to update the state.